service-python/assessclaimdc7101/src/lib/main.py

```diff
@@ -83,6 +83,9 @@ def assess_sufficiency(event: Dict):
                     sufficient = True
             if bp_calculation["recentElevatedBpReadings"] >= 1 and bp_calculation["twoYearsBpReadings"] >= 3:
                 sufficient = True
+        # To DO: remove the following conditional. This should be handled in the camel routes. (HealthEvidenceProcessor)
```

Comment:
IntelliJ highlights the `TODO` string
```suggestion
        # TODO: remove the following conditional. This should be handled in the camel routes. (HealthEvidenceProcessor)
```